### ISSUE TITLE
improved on-boarding for Windows  

### ISSUE THREAD

**Issue by tmm1** [ekzhang/sshx#105]
would be nice to support the one-liner, esp when running inside git-bash (i.e. on azure pipelines with a `bash:` block)

<img width="659" alt="Screenshot 2024-12-28 at 8 53 02 PM" src="https://github.com/user-attachments/assets/6c85371e-9e6f-482e-ac9a-28223062bc06" />

**Comment by ekzhang** [comment]
Makes sense, does it work within git bash if you modify the script? Also would you know how to detect between git bash and other terminals?

**Comment by ekzhang** [comment]
closing due to lack of response
